src/Pipelines/Payloads/ApplyMigrationPayload.php

```diff
@@ -74,6 +76,16 @@ public function getTargetConfiguration()
         return $config;
     }
 
+    /**
+     * @return array
+     */
+    public function getPrefixedTargetConfiguration(): array
+    {
+        $configuration = $this->getTargetConfiguration();
```

Comment:
Do we need a new temp variable everywhere? In the function below `getPrefixedTargetVersionName()` we just use function result directly. Should we make it more consistent?

Reply:
Technically no, I'm just trying to keep the lines short

Reply:
IMHO, I just think it's OK-ish, it would be better if we have consistent style. 😄

Reply:
I changed this